src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -331,6 +332,20 @@ def perform_create(self, serializer):
             token_identifier=token_auth.identifier,
             token_application=token_auth.application,
         )
+        object_type = instance.onderwerpobjectidentificator.get("code_objecttype")
+        if object_type == "zaak":
+            process_cloudevent(
+                type="nl.overheid.zaken.zaak-gelinkt",
+                subject=instance.onderwerpobjectidentificator.get("object_id"),
+                data={
+                    "zaak": f"urn:uuid:{instance.onderwerpobjectidentificator.get('object_id')}",
+                    "linkTo": f"urn:uuid:{instance.uuid}",
```

Comment:
I think for `linkTo` we'll have to use the URL of the OnderwerpObject, because Open Zaak requires a URL to create the ZaakObject: 
<img width="655" height="77" alt="afbeelding" src="https://github.com/user-attachments/assets/3e9a0c42-8434-483d-af93-727354b2e9f5" />

@CharString is this what you had in mind when you specced https://github.com/open-zaak/open-zaak/issues/2251? Since you didn't explicitly mention `linkTo` should be a URL